src/FlightBooking.Core/Entities/Rank.cs

```diff
@@ -0,0 +1,11 @@
+﻿using FlightBookingApp.Core.Common;
+
+namespace FlightBookingApp.Core.Entities
+{
+    public class Rank : BaseAuditableEntity
+    {
+        public string? RankName { get; set; }
+
+        public virtual ICollection<Ticket>? Tickets { get; set; }
```

Comment:
What is the purpose of getting all tickets by its rank? I doubt it will be useful